projects/transformers/models/register_bert_model.py

```diff
@@ -0,0 +1,279 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import re
+
+from torch import nn
+from transformers import (
+    CONFIG_MAPPING,
+    MODEL_FOR_MASKED_LM_MAPPING,
+    MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING,
+    TOKENIZER_MAPPING,
+    BertConfig,
+    BertForMaskedLM,
+    BertForSequenceClassification,
+    BertPreTrainedModel,
+    BertTokenizer,
+    BertTokenizerFast,
+)
+from transformers.models.bert.modeling_bert import BertOnlyMLMHead
+from transformers.utils import logging
+
+# This is the dict of the `models` module. Anything imported from `models`
+# must be contained in this dict. Hence, it will be updated to include
+# the new modules made below so they may be imported.
+from . import __dict__ as __models_dict__
+
+logger = logging.get_logger(__name__)
```

Comment:
Did you meant to redefine logger? My understanding is these logs will not show when calling from run.py if you redefine it here. Let me know if I am wrong about this.

Reply:
I copied this setup from the transformers repo. That's how they do it in most (if not all) files I've seen. I just did a quick test. The logs show up just fine when running `run.py`.

Reply:
Does it picks up up the changes in config applied in run.py?

Reply:
Referring to 
```
        logging.basicConfig(
            format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
            datefmt="%m/%d/%Y %H:%M:%S",
            handlers=[logging.StreamHandler(sys.stdout)],
            level=(logging.INFO if is_main_process(training_args.local_rank)
                   else logging.WARN)
        )
```

Reply:
Reason why I'm asking is I had issues hence changed. I don't clearly remember what was the bug, I think it iwas in the distributed setting. See commit: https://github.com/numenta/nupic.research/commit/d5a944de1cce99881db9f41e0a177edf834294ca.

If you are confident this works fine, we should change it in every file, instead of having half of the files one way, and half of the files the other way.

Reply:
1. It does not use the format `format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s"`
2. I don't know if it works in the distributed setting.
3. Could be could to change it (not sure which is better yet). I can add a TODO if that's alright.

Reply:
Can't we just stick it to the way it is right now instead, I don't see why change it if we don't know if the new way works

Reply:
Sure. It may not make a difference, but I do see the benefit in reducing the potential for errors.